en/resources/tutorials/templating.md

```diff
@@ -31,36 +29,83 @@ You can provide your own Kitura template engine by implementing the `TemplateEng
 
 ---
 
-## Adding a template engine to your Kitura app
+## Setting up
+See [Getting Started](/{{ page.lang }}/starter/gettingstarted.html) for instructions on how to install Kitura's command-line interface.
+
+To use the template engines we first need to setup a Kitura server. Open a terminal and run the following:  
+1. Navigate to Desktop  
+`cd ~/Desktop`
+2. Create a directory  
+`mkdir KituraTemplateExample`
+3. Change to the new directory  
+`cd KituraTemplateExample`
+4. Create the Kitura server  
+`kitura init`
 
-To use a template engine you need to specify a dependency in `Package.swift` for the engine you want to use. You can specify multiple dependencies if you wish to support more than one template engine (e.g. both Mustache and Stencil). In our example we'll add both templates to demonstrate GRMustache and Stencil integration.
+This will take a couple of minutes to complete, `kitura init` downloads Kitura and various supporting packages for monitoring the status of your server and configuration files to allow you to easily deploy to a Docker container, a Kubernetes cluster, or the IBM Cloud. 
```

Comment:
Again spliting up the sentence:
`kitura init` downloads, creates and builds a starter Kitura application and will take a couple of minutes to complete. This includes best-practice implementations of health checking and monitoring as well as configuration files to allow easily deployment to a Docker container, a Kubernetes cluster, or the IBM Cloud."